Build Mutation DeleteContactDetail

ryan-mcneil opened the pull request:
## inTouch-BE PR

### Waffle Project Board

https://waffle.io/ryan-mcneil/inTouch-BE

### Github Issue/Waffle Card #
Closes #39 

### Type of PR
- [x] New Feature
- [ ] Update
- [ ] Bug Fix
- [ ] Configuration

### Summary of changes
 - Added Mutation DeleteContactDetail

### Check List
- [x] Feature Complete
- [ ] Sad Paths Tested
- [ ] All Tests Passing
- [ ] Functionality Documented

### Where should the reviewer start?
- api_v1/schema/contact_detail.py:63

### Additional context?


### Screenshots/Gifs (if applicable)

ryan-mcneil committed:
Build Mutation DeleteContactDetail

wfischer42 reviewed (changes requested):
It seems like the contact_id is an unnecessary requirement here. The request can be validated by checking the contact_id for the specific contact detail, and looking for a contact instance with that ID and the user's user_id.

ryan-mcneil committed:
Update Mutations

Update update and delete to not require contact_id, instead to use the contact_id of the contact_detail_instance

ryan-mcneil commented:
Contact_id has been removed from update and create mutations. the contact_id of the contact_detail_instance is used instead

wfischer42 reviewed (approved):
Looks good!
